signal-element.js

```diff
@@ -50,24 +50,28 @@ function coerce(value) {
 class SignalElement extends HTMLElement {
 	constructor() {
 		super();
-		this.isHTML = this.getAttribute('render') === 'html';
+		this.isHTML = this.getAttribute('render') === 'html' || this.render !== undefined;
 		this.mutation = (state) => state;
 		const initial = this.isHTML
 			? coerce(this.getAttribute('state')) || this.innerHTML
 			: coerce(this.getAttribute('state')) || coerce(this.textContent);
 		this.signal = new Signal.State(initial);
-		this.cleanup = effect(() => this.render());
+		this.cleanup = effect(() => this._render());
 	}
 	connectedCallback() {
-		this.render();
+		this._render();
 	}
 	disconnectedCallback() {
 		this.cleanup();
 	}
-	render() {
+	_render() {
 		const value = this.mutation(this.signal.get());
 		if (this.isHTML) {
-			this.innerHTML = value;
+			if (this.render !== undefined) {
+				this.setHTMLUnsafe(this.render(value));
```

Comment:
I tried this out locally and this if branch actually never runs because `this.render` is not a method on this class and will always be undefined.

This makes sense because `set render(callback)` will define a property on the class and not a method.

There would also be no need to create/call a `render` method anyway because `this.mutation` contains to the render callback function that gets passed to the render property. It would mean we are calling the same function twice.

Reply:
I also did some more testing and was able to recreate the bug you were seeing and it actually just came down to an issue of me forgetting to update `todos.customRenderer` to `todos.render`. 

Any chance that you updated/fixed that property name change at the same time you were implementing this if/else branch in the `_render` method?

Reply:
P.S. Thank you for updating `innerHTML` to `setHTMLUnsafe`!

Reply:
>Any chance that you updated/fixed that property name change at the same time you were implementing
Definitely a possibility as I'm jumping around different branches. 😂

Reply:
Lolol a classic. I've continued to test more and I feel pretty confident that this if branch is not needed so if you could revert it back to the original that would be appreciated :)

Reply:
Reverted!

Reply:
Thank you! Although it's still showing the if/else code on my end? Did you push the change?

Reply:
Ah sorry, just pushed eb897ab. 🚀